### DISCUSSION TITLE
The geohash string was not 6 character

### DISCUSSION THREAD

**Discussion by BassTeQ** [trickypr/bom-weather-docs#7] [Q&A] [answered]
Hi, I'm trying to retrieve information from the location below, but getting an error.

Location Search returned
"geohash":"r1qcnk7","id":"Deer Park-r1qcnk7","name":"Deer Park","postcode":"3023","state":"VIC"

Trying to get the observations using https://api.weather.bom.gov.au/v1/locations/r1qcnk7/observations
Error: {"errors":[{"code":"WEATHER-400","title":"Invalid Geohash","status":"400","detail":"The geohash string was not 6 character."}]}

Any ideas?

**Comment by trickypr** [comment, marked as answer]
When pulling data from the api, BOM only expects a 6 character geohash, whilst some search calls provide a 7 character geohash. Use the first 6 characters and cut of the last one: https://api.weather.bom.gov.au/v1/locations/r1qcnk/observations

**Comment by BassTeQ** [comment]
Great thanks very much!
